en/_posts/2016-10-24-kafka-sasl-ssl.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title:  "Kafka 0.10 with SASL and SSL"
+categories: queue
+author: thbkrkr
+lang: en
+---
+
+This tutorial assumes you already have an OVH account, have subscribed to [OVH Kafka](https://www.runabove.com/dbaas-queue.xml)
```

Comment:
I'd add a link to account creation page: "If you do not have an OVH account, you may create one here ..."